relay-profiling/src/sample_v2.rs

```diff
@@ -0,0 +1,221 @@
+use std::collections::{BTreeMap, HashSet};
+use std::ops::Range;
+
+use chrono::{DateTime, Utc};
+use itertools::Itertools;
+use serde::{Deserialize, Serialize};
+
+use crate::error::ProfileError;
+use crate::measurements::Measurement;
+use crate::sample_v1::{DebugMeta, Frame, ThreadMetadata, Version};
+
+const MAX_PROFILE_CHUNK_DURATION_MS: f64 = 10000f64;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ProfileMetadata {
+    chunk_id: String,
+    profiler_id: String,
+
+    #[serde(skip_serializing_if = "Option::is_none")]
+    debug_meta: Option<DebugMeta>,
+
+    #[serde(skip_serializing_if = "Option::is_none")]
+    environment: Option<String>,
+    platform: String,
+    release: String,
+
+    timestamp: DateTime<Utc>,
+    version: Version,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+struct Sample {
+    timestamp_ms: f64,
```

Comment:
Is this a relative or absolute timestamp?

Reply:
They are all unix timestamps, so absolute.